.gitignore

```diff
@@ -50,6 +50,7 @@ storage/shared_storage/*
 storage/sitemaps/*
 tmp/*
 deps/*
+metadata.json
```

Comment:
we're going to need some docs to explain what this metadata.json file is (and a less vague name) - I can see anywhere it's written, only read.

Reply:
I've added some doc updates. LMKWYT.

Reply:
Can we have a more specific name?  metadata.json could be anything.

Reply:
Done.